src/api/logger.js

```diff
@@ -0,0 +1,57 @@
+import winston from "winston";
+
+// Define the severity levels
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+};
+
+/*
+ If the server was on development mode, show all the log levels.
+ If the server was on production mode, show only warn and error messages.
+*/
+const level = () => {
+  const env = process.env.NODE_ENV || 'development'
+  const isDevelopment = env === 'development'
+  return isDevelopment ? 'debug' : 'warn'
+};
+
+const colors = {
+  error: 'red',
+  warn: 'yellow',
+  info: 'green',
+  http: 'magenta',
+  debug: 'white',
+};
+
+// Show different color for different severity levels.
+winston.addColors(colors);
+
+const format = winston.format.combine(
+  // Add the message timestamp
+  winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss:ms' }),
+  // Colorize the messages
+  winston.format.colorize({ all: true }),
+  // Define the format of the message
+  winston.format.printf(
+    (info) => `${info.timestamp} ${info.level}: ${info.message}`,
+  ),
+);
+
+
+const transports = [
+  // Print the messages in console.
+  new winston.transports.Console(),
+];
```

Comment:
Is there any options to print the messages to somewhere besides the `console`?

Reply:
We can print the log in files. And there are also option for third party service. I though console would be sufficient for now. I will add file besides console.